swift_scality_backend/diskfile.py

```diff
@@ -481,6 +482,16 @@ def open(self):
         if metadata is None:
             raise swift.common.exceptions.DiskFileDeleted()
         self._metadata = metadata or {}
+
+        try:
+            x_delete_at = int(self._metadata['X-Delete-At'])
+        except KeyError:
+            pass
+        else:
```

Comment:
Swift's native `DiskFile` implementation also handles `ValueError` on `int` conversion failure. Anything we can do here?